Tools/environment_install/install-prereqs-arch.sh

```diff
@@ -26,7 +26,7 @@ BASE_PKGS="base-devel ccache git gsfonts tk wget gcc"
 SITL_PKGS="python-pip python-setuptools python-wheel wxpython opencv python-numpy python-scipy"
 PX4_PKGS="lib32-glibc zip zlib ncurses"
 
-PYTHON_PKGS="future lxml pymavlink MAVProxy pexpect argparse matplotlib pyparsing geocoder pyserial empy"
+PYTHON_PKGS="future lxml pymavlink nunavut MAVProxy pexpect argparse matplotlib pyparsing geocoder pyserial empy"
```

Comment:
can we add this as submodule and call it directly?

Reply:
fixed